core/src/commonMain/kotlin/com/powersync/sync/SyncStream.kt

```diff
@@ -174,6 +180,7 @@ internal class SyncStream(
             headers {
                 append(HttpHeaders.Authorization, "Token ${credentials.token}")
                 append("User-Id", credentials.userId ?: "")
```

Comment:
While you're working on headers, could you also remove the "User-Id" one in both places? It is never used by the service, and not included by other SDKs.

Reply:
Will do